contracts/liquidity/Liquidity.sol

```diff
@@ -378,6 +369,61 @@ contract Liquidity is
 		}
 	}
 
+	function _processDirectWithdrawal(
+		WithdrawalLib.Withdrawal memory withdrawal_
+	) internal {
+		TokenInfo memory tokenInfo = getTokenInfo(withdrawal_.tokenIndex);
+
+		bool result = true;
```

Comment:
I think it would be better to set the initial value to true, and then set it to false once you've entered the catch block.

@kbizikav